src/encryptdecrypt/Main.java

```diff
@@ -0,0 +1,27 @@
+package encryptdecrypt;
+
+import encryptdecrypt.cryptors.AbstractCryptor;
+import encryptdecrypt.cryptors.ConcreteCryptor;
+import encryptdecrypt.parsers.CliParser;
+import encryptdecrypt.strategy.ShiftAlgorithm;
+import encryptdecrypt.strategy.UnicodeAlgorithm;
+
+public class Main {
+
+    public static void main(String[] args) {
+        CliParser parser = new CliParser(args);
+
+        AbstractCryptor cryptor = new ConcreteCryptor(
+                parser.optionValueOrDefault("-alg", "shift")
+                        .equals("shift") ? new ShiftAlgorithm() : new UnicodeAlgorithm(),
+
+                parser.optionValueOrDefault("-mode", "enc"),
+                parser.optionValueOrDefault("-data", ""),
```

Comment:
Does ConcreteCryptor use only once?

Reply:
Just like in the task description
- [X] Done